packages/g/gridjs.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "gridjs",
+  "filename": "gridjs.production.min.js",
+  "description": "Grid.js is a Free and open-source HTML table plugin written in TypeScript. It works with most JavaScript frameworks, including React, Angular, Vue and VanillaJs.",
+  "keywords": [
+    "gridjs",
+    "grid",
+    "table",
+    "sort",
+    "filter",
+    "search"
+  ],
+  "homepage": "https://github.com/grid-js/gridjs",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/grid-js/gridjs.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "gridjs",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*",
+          "themes/*"
```

Comment:
```suggestion
          "*.@(js?(.map)|d.ts)",
          "theme/*.css?(.map)"
```
Let's restrict the wildcards, so we're only pulling in the file types we need. Also `themes` -> `theme`, based on the content of the package.

[Glob test](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%7B%2A.%40%28js%3F%28.map%29%7Cts%29%2Ctheme%2F%2A.css%3F%28.map%29%7D&matches=false&tests=%2F%2F%20Imported%20from%20gridjs%20NPM%20package&tests=LICENSE&tests=dist%2Ftheme%2Fmermaid.css&tests=dist%2Ftheme%2Fmermaid.min.css&tests=dist%2Fgridjs.development.es.js&tests=dist%2Fgridjs.development.js&tests=dist%2Fgridjs.production.es.min.js&tests=dist%2Fgridjs.production.min.js&tests=package.json&tests=dist%2Fgridjs.development.es.js.map&tests=dist%2Fgridjs.development.js.map&tests=dist%2Fgridjs.production.es.min.js.map&tests=dist%2Fgridjs.production.min.js.map&tests=dist%2Ftheme%2Fmermaid.css.map&tests=README.md&tests=dist%2Fsrc%2Fview%2Fbase%2Factions.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsearch%2Factions.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsort%2Factions.d.ts&tests=dist%2Fsrc%2Futil%2Farray.d.ts&tests=dist%2Fsrc%2Fpipeline%2Ftransformer%2FarrayToTabular.d.ts&tests=dist%2Fsrc%2Fbase.d.ts&tests=dist%2Fsrc%2Fview%2Fbase.d.ts&tests=dist%2Fsrc%2Fcell.d.ts&tests=dist%2Fsrc%2Futil%2FclassName.d.ts&tests=dist%2Fsrc%2Fconfig.d.ts&tests=dist%2Fsrc%2Fview%2Fcontainer.d.ts&tests=dist%2Fsrc%2Futil%2Fdebounce.d.ts&tests=dist%2Fsrc%2Futil%2Fdispatcher.d.ts&tests=dist%2Fsrc%2Fi18n%2Fen_US.d.ts&tests=dist%2Fsrc%2Futil%2FeventEmitter.d.ts&tests=dist%2Fsrc%2Fevents.d.ts&tests=dist%2Fsrc%2Fview%2Fevents.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Fevents.d.ts&tests=dist%2Fsrc%2Fview%2FfooterContainer.d.ts&tests=dist%2Fsrc%2Futil%2FgetConfig.d.ts&tests=dist%2Fsrc%2Fpipeline%2Ffilter%2FglobalSearch.d.ts&tests=dist%2Fsrc%2Fgrid.d.ts&tests=dist%2Fsrc%2Fheader.d.ts&tests=dist%2Fsrc%2Fview%2FheaderContainer.d.ts&tests=dist%2Fsrc%2Futil%2Fhtml.d.ts&tests=dist%2Fsrc%2Fview%2FhtmlElement.d.ts&tests=dist%2Fsrc%2Futil%2Fid.d.ts&tests=dist%2Findex.d.ts&tests=dist%2Fsrc%2Ftheme%2Fmermaid%2Findex.d.ts&tests=dist%2Fsrc%2Fi18n%2Flanguage.d.ts&tests=dist%2Fsrc%2Futil%2Flog.d.ts&tests=dist%2Fsrc%2Fstorage%2Fmemory.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2FmessageRow.d.ts&tests=dist%2Fsrc%2Fpipeline%2Fsort%2Fnative.d.ts&tests=dist%2Fsrc%2Fpipeline%2Flimit%2Fpagination.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fpagination.d.ts&tests=dist%2Fsrc%2Fpipeline%2Fpipeline.d.ts&tests=dist%2Fsrc%2Fpipeline%2FpipelineUtils.d.ts&tests=dist%2Fsrc%2Fplugin.d.ts&tests=dist%2Fsrc%2Fpipeline%2Fprocessor.d.ts&tests=dist%2Fsrc%2Frow.d.ts&tests=dist%2Fsrc%2Foperator%2Fsearch.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsearch%2Fsearch.d.ts&tests=dist%2Fsrc%2Fpipeline%2Finitiator%2Fserver.d.ts&tests=dist%2Fsrc%2Fpipeline%2Fsort%2Fserver.d.ts&tests=dist%2Fsrc%2Fstorage%2Fserver.d.ts&tests=dist%2Fsrc%2Fpipeline%2Ffilter%2FserverGlobalSearch.d.ts&tests=dist%2Fsrc%2Fpipeline%2Flimit%2FserverPagination.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Fshadow.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsort%2Fsort.d.ts&tests=dist%2Fsrc%2Fpipeline%2Fextractor%2Fstorage.d.ts&tests=dist%2Fsrc%2Fstorage%2Fstorage.d.ts&tests=dist%2Fsrc%2Fpipeline%2Ftransformer%2FstorageResponseToArray.d.ts&tests=dist%2Fsrc%2Fstorage%2FstorageUtils.d.ts&tests=dist%2Fsrc%2Fview%2Fbase%2Fstore.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsearch%2Fstore.d.ts&tests=dist%2Fsrc%2Fview%2Fplugin%2Fsort%2Fstore.d.ts&tests=dist%2Fsrc%2Futil%2Fstring.d.ts&tests=dist%2Fsrc%2Futil%2Ftable.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Ftable.d.ts&tests=dist%2Fsrc%2Ftabular.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Ftbody.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Ftd.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Fth.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Fthead.d.ts&tests=dist%2Fsrc%2Fview%2Ftable%2Ftr.d.ts&tests=dist%2Fsrc%2Ftypes.d.ts&tests=dist%2Fsrc%2Futil%2Fwidth.d.ts)